setup.py

```diff
@@ -1,130 +0,0 @@
-    long_description=read_readme(),
-    long_description_content_type="text/markdown",
-    url="https://github.com/yourusername/swaybgplus",
-    project_urls={
-        "Bug Reports": "https://github.com/yourusername/swaybgplus/issues",
-        "Source": "https://github.com/yourusername/swaybgplus",
-        "Documentation": "https://github.com/yourusername/swaybgplus/blob/main/README.md",
-    },
-    packages=find_packages(),
-    classifiers=[
-        "Development Status :: 4 - Beta",
-        "Environment :: X11 Applications :: GTK",
-        "Environment :: Wayland",
-        "Intended Audience :: End Users/Desktop",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: POSIX :: Linux",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11",
-        "Topic :: Desktop Environment",
-        "Topic :: Multimedia :: Graphics",
-        "Topic :: System :: Systems Administration",
-    ],
-    keywords="sway wayland background wallpaper monitor multi-monitor orientation",
-    python_requires=">=3.6",
-    install_requires=[
-        "Pillow>=7.0.0",
-        "PyGObject>=3.30.0",
-    ],
-    extras_require={
-        "dev": [
-            "pytest>=6.0",
-            "black>=21.0",
-            "flake8>=3.8",
-        ],
-    },
-    entry_points={
-        "console_scripts": [
-            "swaybgplus=swaybgplus_cli:main",
-            "swaybgplus-cli=swaybgplus_cli:main",
-            "swaybgplus-gui=swaybgplus_gui:main",
-        ],
-    },
-    scripts=[
-        "swaybgplus_cli.py",
-        "swaybgplus_gui.py",
-    ],
-    data_files=[
-        ("share/applications", ["swaybgplus.desktop"]),
-        ("share/doc/swaybgplus", ["README.md"]),
-        ("share/man/man1", ["swaybgplus.1"]) if os.path.exists("swaybgplus.1") else [],
-    ],
-    include_package_data=True,
-    zip_safe=False,
-    platforms=["linux"],
-)
-
-# Post-installation message
-def post_install():
-    print("\n" + "="*60)
-    print("🎉 SwayBG+ installed successfully!")
-    print("="*60)
-    print("📱 CLI Usage:")
-    print("   swaybgplus image.jpg --mode stretched")
-    print("   swaybgplus --orientation DP-1:90")
-    print()
-    print("🖥️  GUI Usage:")
-    print("   swaybgplus-gui")
-    print()
-    print("📖 Documentation:")
-    print("   man swaybgplus")
-    print("   /usr/share/doc/swaybgplus/README.md")
-    print()
-    print("🔧 First time setup:")
-    print("   1. Make sure Sway and swaybg are installed")
-    print("   2. Run: swaybgplus-gui to configure monitors")
-    print("   3. Load an image and apply backgrounds")
-    print("="*60)
```

Comment:
There is no equivalent in `pyproject.toml` that I am aware of. Might want to move this into the PKGBUILD instead.